lua/org-bullets.lua

```diff
@@ -72,20 +73,22 @@ local markers = {
   end,
   -- Checkboxes [x]
   expr = function(str, conf)
-    local text = { str, "OrgTODO" }
     local symbols = conf.symbols.checkboxes
     if str:match("X") then
-      text = symbols.done
-    elseif str:match("%-") then
-      text = symbols.cancelled
-    else
-      text = symbols.todo
+      return { { "[", "OrgTSCheckboxChecked" }, symbols.done, { "]", "OrgTSCheckboxChecked" } }
+    elseif str:match("-") then
+      return {
+        { "[", "OrgTSCheckboxHalfChecked" },
+        symbols.half,
+        { "]", "OrgTSCheckboxHalfChecked" },
+      }
+      --[[ elseif str:match(" ") then
+      return { { "[", "OrgTSCheckbox" }, symbols.undone, { "]", "OrgTSCheckbox" } } ]]
     end
-    return { { "[", "NonText" }, text, { "]", "NonText" } }
```

Comment:
Similar to the other comment, a lot of this is stuff I'm still working on locally, so this part of the change should be reverted as well as I'm tweaking this locally as well. The rest of the change re. the highlight names is fine though

Reply:
Orgmode highlight - [DOCS.md](https://github.com/nvim-orgmode/orgmode/blob/master/DOCS.md#colors)